src/utp_stream.cpp

```diff
@@ -1888,6 +1889,7 @@ bool utp_socket_impl::send_pkt(int const flags)
 		if (payload_size)
 		{
 			p = static_cast<packet*>(malloc(sizeof(packet) + effective_mtu));
+			memset(p, 0, sizeof(packet));
```

Comment:
would it make more sense to initialize ``mtu_probe`` to false here? it seems unnecessarily expensive to clear the entire buffer, immediately before a ``memcpy()``.